src/rollbar.js

```diff
@@ -260,6 +261,15 @@ Rollbar.prototype._addTracingInfo = function (item) {
   }
 };
 
+Rollbar.prototype._getEnabledTrigger = function (triggers, level) {
+  for (const t of triggers) {
+    if (t.level?.includes(level)) {
+      return t;
+    }
+  }
+  return null;
+};
```

Comment:
Does this need to be a function? Seems pretty straight-forward/self-documenting at the point of use:
```ts
const enabledTrigger = this._getEnabledTrigger(triggers, item.level);
```
Becomes
```ts
const enabledTrigger = triggers.find(t => t.level?.includes(level)) || null;
```
The code literally says "find the trigger with the levels that includes some `level`."

Reply:
There will be additional filters.